src/ballet/chacha20/test_fd_chacha20_rng.c

```diff
@@ -0,0 +1,60 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <assert.h>
+#include "fd_chacha20.h"
+
+#define TEST_KEY_SIZE FD_CHACHA20_KEY_SIZE
+#define TEST_NONCE_SIZE FD_CHACHA20_NONCE_SIZE
+
+/* Test fd_chacha20_rng_init function */
+void test_fd_chacha20_rng_init()
+{
+  unsigned char key[TEST_KEY_SIZE];
```

Comment:
I don't know if we have a public comprehensive style guide yet, but these should be `uchar`

Reply:
Looks like `uchar` type is not directly available In OpenSSL

I'm getting the following error while trying to use it -> `error: unknown type name 'uchar'; did you mean 'char'?`

Reply:
thanks! just solved this by adding `#include "../fd_ballet_base.h"` , which uses `fd_util.h` under the hood 👍🏻